core/common/src/main/java/alluxio/exception/BlockDoesNotExistRuntimeException.java

```diff
@@ -21,14 +23,15 @@
 public class BlockDoesNotExistRuntimeException extends AlluxioRuntimeException {
```

Comment:
Not really in the scope of this PR: Technically this should extend `NotFoundRuntimeException`. (Or even better, just remove this and use `NotFoundRuntimeException` directly.